src/lib/jsonParseV2/parseEntity.cpp

```diff
@@ -79,17 +79,18 @@ std::string parseEntity(ConnectionInfo* ciP, Entity* eP, bool eidInURL)
     {
       LM_W(("Bad Input (No entity id specified"));
       eP->errorCode.fill("ParseError", "no entity id specified");
-      ciP->httpStatusCode = SccBadRequest;;
+      ciP->httpStatusCode = SccBadRequest;
+
       return eP->render(ciP, EntitiesRequest);
     }
   }
-  else if (document.Empty()) 
```

Comment:
Maybe we should add a comment here like "// We have experienced crash problems with Empty(), so ObjectEmpty() is used instead" to avoid that in the future we forget this and somebody changes back to Empty()....